tests/native/src/test/scala/org/threeten/bp/zone/TestStandardZoneRulesNegativeTZDB.scala

```diff
@@ -0,0 +1,95 @@
+/*
```

Comment:
We could maybe remove this as this wasn't created by Stephen or Michael

Reply:
I just copied it, so the original authors' names should remain.